pkgs/top-level/all-packages.nix

```diff
@@ -4714,6 +4714,8 @@ with pkgs;
 
   endlessh = callPackage ../servers/endlessh { };
 
+  ericw-tools = callPackage ../applications/misc/ericw-tools { };
```

Comment:
There are many changes here which seem to be totally unrelated.

Reply:
Let me rebase again to master. Seems like something went wrong.